lib/fennec/evaluator.ex

```diff
@@ -2,16 +2,15 @@ defmodule Fennec.Evaluator do
   @moduledoc false
 
   alias Jerboa.Params
+  alias Fennec.TURN
 
-  @spec service(Params.t, map) :: Params.t | :void
-  def service(p, changes) do
+  @spec service(Params.t, map, TURN.t) :: {Params.t, TURN.t} | :void
+  def service(p, changes, turn_state) do
```

Comment:
Ok then. For me it's a some kind of client's connection state. For now it holds current IP/Port of the client connection. Co maybe it should be `conn_state`? @mentels what do you think about that?